services/community-site/src/pages/Redeem/redeem.scss

```diff
@@ -1,3 +1,15 @@
+.container {
```

Comment:
You need to add these in a `.redeem` wrapper or something. Or else all of these classes will be applied to everywhere.

Reply:
I'm sorry but I don't really get what you mean by this. Aren't these classes only on the scope of the JSX file it gets imported into?